3p/integration.js

```diff
@@ -56,6 +56,7 @@ import {adpicker} from '#ads/vendors/adpicker';
 import {adplugg} from '#ads/vendors/adplugg';
 import {adpon} from '#ads/vendors/adpon';
 import {adpushup} from '#ads/vendors/adpushup';
+import {andbeyond} from '#ads/vendors/andbeyond';
```

Comment:
You no longer need the changes in this file, sorry this likely changed while this PR was in flight.